src/general/OptionsMenu.cs

```diff
@@ -1002,6 +1002,11 @@ private void OnTutorialsOnNewGameToggled(bool pressed)
     private void OnCheatsToggled(bool pressed)
     {
         Settings.Instance.CheatsEnabled.Value = pressed;
+        if (!pressed)
+        {
+            CheatMenu.Instance.DisableAllCheats();
+            CheatMenu.Instance.IsMenuOpen = false;
```

Comment:
Shouldn't this just be Hide?

Reply:
The IsMenuOpen setter internally calls hide

Reply:
I'd prefer that this didn't have access to also showing the thing, would be a cleaner interface here to just say "go away". Though if the CheatManager is added in between here, I wouldn't mind that directly using an IsMenuOpen property like this.